CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+## Expressions
+- Removed `UnaryOperator.OnNewOperand` and `OnLostOperand`.  This were part of a broken pattern of using unary expressions (and were not present on Binary/Ternary/QuaternaryOperator). You will need to extend Expression and implement `Subscribe`, using `ExpressionListener` as a way to capture the correct functionality. It was already broken before, this forces a fix to be made.
```

Comment:
Can't we mark this as obsolete for a few releases first so code implementing this will still compile, but give a warning?

Reply:
Is it possible, but it's relatively tricky to manage, and would only provide a runtime warning. I need to pass a deprecated flag to the base-class that changes how it works, basically breaking out into a different model (Which might be a tad bit broken -- it was broken before as well)

Reply:
I'll push my attempt as a separate commit, then you can see if it's a good idea to retain.